locales/en-US/app.ftl

```diff
@@ -43,6 +43,73 @@ AppViewRouter--error-message-from-url =
 AppViewRouter--route-not-found--home =
     .specialMessage = The URL you tried to reach was not recognized.
 
+## CallNodeContextMenu
+## This is used as a context menu for the Call Tree, Flame Graph and Stack Chart
+## panels.
+
+CallNodeContextMenu--transform-merge-function = Merge function
+    .title =
+        Merging a function removes it from the profile, and assigns its time to the
+        function that called it. This happens anywhere the function was called in
+        the tree.
+CallNodeContextMenu--transform-merge-call-node = Merge node only
+    .title =
+        Merging a node removes it from the profile, and assigns its time to the
+        function's node that called it. It only removes the function from that
+        specific part of the tree. Any other places the function was called
```

Comment:
The last sentence seems off? "was called from" or something else completely?

Reply:
Yeah, it should  be "was called from". Fixed it.